### PR TITLE
feat: Creator/Curator/User statistics

### PR DESCRIPTION
**Pull request by FedeMadoery**
# Description

Add base elements for user statistics, and implements the creator, curator and user statistics.

### CODE REVIEW

File: src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+import SafeSuspense from '@/src/components/helpers/SafeSuspense'
+import useStatisticVisibility from '@/src/hooks/nextjs/useStatisticsVisibility'
+import StatisticsHeader from '@/src/pagePartials/profile/statistics/addons/StatisticsHeader'
+import { StatisticVisibilityToggle } from '@/src/pagePartials/profile/statistics/addons/StatisticsVisibilityToggle'
+import CreatorStatisticContent from '@/src/pagePartials/profile/statistics/creator/CreatorStatisticContent'
+
+export enum CreatorStatistic {
+  fee = 'Fees collected per badge',
+  amount = 'Amount of badge types created ',
```
Comment (Agupane) [inline src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx:17]: Amount of badge models created

File: src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  minters = 'Amount of users that minted badges types',
```
Comment (Agupane) [inline src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx:18]: Amount of users that minted badge models

File: src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  minted = 'Amount of badges types minted',
```
Comment (Agupane) [inline src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx:19]: Amount of badge models minted

**Review by Agupane — CHANGES_REQUESTED** [review]

File: src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx (hunk 2 above)
Comment (ramabit) [reply src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx:18]: well, in reality the user mints badges, not badge models. So, this should be: "Amount of users that minted badges with the created badge models" or similar

Comment (Agupane) [reply src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx:18]: but that would be extremelly long and will break the UI 
image

File: src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx (hunk 3 above)
Comment (ramabit) [reply src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx:19]: Similar as above, this should be: "Amount of badges minted using created badge models"

File: src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx (hunk 2 above)
Comment (ramabit) [reply src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx:18]: we can for example remove the "Amount of" and allow in the UI a second line if needed

Comment (Agupane) [reply src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx:18]: Yeah that would be good, removing "Amount of"

Comment (ramabit) [reply src/pagePartials/profile/statistics/creator/CreatorStatistics.tsx:18]: another option is to keep it keep like: "Users that minted badge model" and a tooltip explaining the larger description

**Review by Agupane — APPROVED** [review]

**Pull request merged**